presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestRemoveRedundantTopN.java

```diff
@@ -70,6 +70,7 @@ public void test()
     public void testZeroTopN()
     {
         tester().assertThat(new EvaluateZeroTopN())
+                //tester().assertThat(new RemoveRedundantTopN())
```

Comment:
why is this commented out?

Reply:
comment out the incorrect class to test, remove comment